### ISSUE TITLE
[question] How to see log of launcher?

### ISSUE THREAD

**Issue by jadekler** [karma-runner/karma#2465]
e.x. karma-phantomjs-launcher / karma-chrome-launcher / karma-script-launcher

**Comment by jadekler** [comment]
Am I right in assuming that [this piece of code](https://github.com/karma-runner/karma/blob/master/lib/launchers/process.js#L140-L148) means that there's no way to see launcher output?

**Comment by jadekler** [comment]
I was able to replace that code with the following to make it work, but I'd really prefer to have a flag to do this. Would it make sense to PR this?

```
  var spawnWithoutOutput = function () {
      var proc = spawn.apply(null, arguments)
      log.info("hello world")
      proc.stdout.pipe(process.stdout)
      proc.stderr.pipe(process.stderr)

      return proc
    }

    ProcessLauncher.call(launcher, spawnWithoutOutput, require('../temp_dir'), timer)
```

**Comment by maksimr** [comment]
This was introduced in this https://github.com/karma-runner/karma/pull/2267 Pull request

**Comment by maksimr** [comment]
@jadekler 

> Would it make sense to PR this?

Yes. Maybe will be better output browser log by default and hide behind the falg feature from Pull Request #2267 . Because case from #2267 

1. Rare
2. Hide errors from the browser and I think it's not so obviously for user, more obviously look at flag which resume browser log (IMHO)

@dignifiedquire what do you think?

**Comment by dignifiedquire** [comment]
It should be possible to pipe the output from the spawn to the console, without creating the buffer issue. It might be interesting to try out for spawning https://github.com/sindresorhus/execa which I have made good experiences in the past.
